### DISCUSSION TITLE
Can't click() on some web page since 126.2.70

### DISCUSSION THREAD

**Discussion by tifish** [cefsharp/CefSharp#4913] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

126.2.180

### Operating System

Windows 11

### Architecture

x64

### .Net Version

.Net 8.0

### Implementation

WinForms

### Reproduction Steps

1. Load web page: https://store.epicgames.com/en-US/download
2. Call `EvaluateScriptAsync()` with javascript: `document.evaluate("//a[contains(@href, '.msi')]", document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue.click()`

### Expected behavior

Should start downloading.

### Actual behavior

The downloading won't start since 126.2.70, including 126.2.180 and 127.3.50.

### Regression?

125.0.210 and below is OK.

### Does this problem also occur in the CEF Sample Application

Not Tested

**Comment by amaitland** [comment]
Converting to discussion as this is very unlikely a bug in `CefSharp`.

Have you confirmed the javascript runs in `DevTools` console? Have you confirmed that it also runs in `Chrome` at the **exact** same version?

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#javascript-debugging

Every new version of `Chromium` brings a different `V8` engine with different rules and restrictions.

> 2\. Call `EvaluateScriptAsync()`

Did you check the response? What was the error?

If you aren't returning a value then switch to `ExecuteScriptAsync` which directly calls the underlying `Chromium` method.

**Comment by MyzLee** [comment]
I've got the same problems. Based on the initial ticket and the descriptions here, I have created an example. Version 125.0.210 is used here. If you switch to ChromeRuntime in the constructor of the MainWindow, only simple click events are working (switch to 'Terms of Service'). The 'complex' click events are no longer triggered and the LifeSpanHandler no longer works as expected. In addition, JS alerts then only work with activated DevTools. This behavior is still unchanged in the latest version 130.1.90.

[WpfApp1.zip](https://github.com/user-attachments/files/17660336/WpfApp1.zip)

**Reply by amaitland** [reply]
> Converting to discussion as this is very unlikely a bug in CefSharp.

As I said this is unlikely a bug in `CefSharp`.

CEF has it's own issue tracker at https://github.com/chromiumembedded/cef/issues

> In addition, JS alerts then only work with activated DevTools.

This is an upstream issue

**Reply by amaitland** [reply]
If the javascript triggers a `window.open` then you might want to try disabling the popup blocker that's now on by default.

https://github.com/cefsharp/CefSharp/issues/4940#issuecomment-2382303302

**Reply by MyzLee** [reply]
Thank you very much, that has solved my problems!
